app/Http/Requests/StoreMultipleInventoryItemsInEvent.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreMultipleInventoryItemsInEvent extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'events' => 'array'
```

Comment:
MUST: Die Validierung ist nicht ausreichend. Hier müssen mindestens alle erforderlichen Felder geprüft werden. Das fliegt dir sofort um die Ohren sobald irgendein Key nicht da ist (im guten Fall), im schlechten Fall haben wir kaputte Daten in der DB

MUST: Das muss in die Domäne, das liegt noch im App Ordner

Ein ungefähres Beispiel findest du in artwork/Modules/InventoryManagement/Http/Requests/Export/CreateInventoryManagementExportRequest.php